tests/python/test_transformation.py

```diff
@@ -38,15 +38,15 @@ def test_transformation():
 
     # load a 1 GeV electron beam with an initial
     # unnormalized rms emittance of 2 nm
-    energy_MeV = 1e3  # reference energy
-    energy_gamma = energy_MeV / 0.510998950
+    kin_energy_MeV = 1e3  # reference energy
+    energy_gamma = kin_energy_MeV / 0.510998950 + 1.0
```

Comment:
This is a tiny bug fix and has no big influence for the 1 GeV reference particle energy used here.
Mainly added for clarity for readers.